### DISCUSSION TITLE
Reorder forms in tasks

### DISCUSSION THREAD

**Discussion by RolandvonBorstel** [sadieblue/agilityblue.workspace-scripts#11] [Q&A] [answered]
Is there any way to reorder forms in a task using Scripts or API?

**Comment by bmward80** [comment, marked as answer]
Hi @RolandvonBorstel, 

Task forms don't have any kind of sort order field, which means they can only be rendered in the order they are created by their id within the forms array. Technically, you could copy out the information from an existing task form into a new task form, zero/null out the id fields of all the child items, add it as a new form, then remove the previous version... so it's possible to achieve the effect, but it's not a true re-sort of the existing task forms.

Good question,
Brandon

**Reply by RolandvonBorstel** [reply]
Hi Brandon. Thanks for the insight. Is there a command to remove a form from a task? I have only seen Add-FormsToTask but not a remove. command.

**Reply by bmward80** [reply]
Roland, there is no special command to remove forms. You should be able to just remove a form directly from the Forms array on a task and then save the task back with `Set-Task`. The system should take care of removing the form at that point.

Brandon
